frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/Range.java

```diff
@@ -1,49 +1,52 @@
 // Copyright (c) Microsoft Corporation
-// All rights reserved. 
+// All rights reserved.
 //
 // MIT License
 //
-// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated 
-// documentation files (the "Software"), to deal in the Software without restriction, including without limitation 
-// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and 
```

Comment:
>copy [](start = 22, length = 4)

Do not format the License

Reply:
space.

---
In reply to: [173102461](https://github.com/Microsoft/pai/pull/318#discussion_r173102461) [](ancestors = 173102461)

Reply:
Still the same 

---
In reply to: [174454511](https://github.com/Microsoft/pai/pull/318#discussion_r174454511) [](ancestors = 174454511,173102461)